native-auth/config.md

```diff
@@ -9,3 +9,41 @@ dto NativeAuthClientConfig:
     gateway_url?: string;
     extra_request_headers?: dict[string, Any];
 ```
+
+## NativeAuthServerConfig
+
+```
+dto NativeAuthServerConfig:
+    // The endpoint from where the current block information will be fetched upon validation.
```

Comment:
Maybe here would also work something like this
`API endpoint to fetch the current block information for validation.`